laundry-management-service/laundrymanagement-app/src/main/java/com/microservices/laundrymanagement/configuration/OrderServiceBeansConfig.java

```diff
@@ -3,40 +3,40 @@
 import com.microservices.laundrymanagement.dto.DetailSubmissionDto;
 import com.microservices.laundrymanagement.dto.OrderSubmissionDto;
 import com.microservices.laundrymanagement.entity.OrderEntity;
-import com.microservices.laundrymanagement.entity.OrderStatus;
 import org.modelmapper.Converter;
 import org.modelmapper.ModelMapper;
 import org.modelmapper.PropertyMap;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import static org.modelmapper.config.Configuration.AccessLevel.PRIVATE;
+
 @Configuration
 public class OrderServiceBeansConfig {
 
     @Bean
     ModelMapper modelMapper() {
         ModelMapper modelMapper = new ModelMapper();
 
+        modelMapper.getConfiguration()
+                .setFieldMatchingEnabled(true)
+                .setFieldAccessLevel(PRIVATE);
+
+        Converter<OrderSubmissionDto, OrderEntity> toOrderEntityPostConverter = context -> {
```

Comment:
Think we can move this to field. @AfanasievYuriy how do you feel it it worth to create separate bean like OrderMapper?